lib/cisco_node_utils/cmd_ref/acl.yaml

```diff
@@ -0,0 +1,29 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
+  config_get_token: '/^<afi> access-list <acl_name>$/'
+  config_set: '<afi> access-list <acl_name>'
+
+acl:
+  config_get_token: '/^<afi> access-list (\S+)$/'
+  config_set: '<state> <afi> access-list <acl_name>'
+                      
+all_acl:
```

Comment:
Nit: To keep with our coding convention I would prefer to rename this tag `all_acls`